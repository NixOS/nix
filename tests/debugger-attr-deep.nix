let 
  testattrs = { a = 1; b = wat 5; };
  wat = arg : arg + testattrs.b;
  foo = arg : arg2 : arg + arg2;
  
in
testattrs