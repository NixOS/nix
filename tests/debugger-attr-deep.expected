[34;1m0[0m: while evaluating the attribute '[35;1mb[0m'
[34;1m1[0m: while calling [35;1m'wat'[0m
[34;1m2[0m: while evaluating the attribute '[35;1mb[0m'
[34;1m3[0m: [31;1merror: [0minfinite recursion encountered
